source/features/filter-comments-by-you.tsx

```diff
@@ -21,6 +21,7 @@ function init() {
 
 features.add({
 	id: 'filter-comments-by-you',
+	description: 'Search for issues and PRs with the `Everything commented by you filter`',
```

Comment:
@bfred-it Would this look good after backticks are parsed?

It's not exactly code, per se. Or can bold be used in cases like these?

Reply:
It's how we've been marking them up in the readme: https://github.com/sindresorhus/refined-github/blob/0b6c7330c21f8570519143ad3d0decc328d4db47/readme.md#L185

@sindresorhus thoughts? Should they be in _italics_ (in the readme too)?

Reply:
@sindresorhus any comments on this? I would like to close this PR as soon as possible.

Reply:
Yes, let's go with italic.